libs/wire-api/src/Wire/API/Error/Galley.hs

```diff
@@ -113,6 +142,18 @@ type instance MapError 'ConvAccessDenied = 'StaticError 403 "access-denied" "Con
 
 type instance MapError 'MLSNonEmptyMemberList = 'StaticError 400 "non-empty-member-list" "Attempting to add group members outside MLS"
 
+type instance MapError 'DuplicateMLSPublicKey = 'StaticError 400 "mls-duplicate-public-key" "MLS public key for the given signature scheme already exists"
+
+type instance MapError 'KeyPackageRefNotFound = 'StaticError 404 "mls-key-package-ref-not-found" "A referenced key packages could not be mapped to a known client"
```

Comment:
A mistake in plurality: packages -> package.

Reply:
This and the other mistakes below were fixed in later commits.